src/service/index.js

```diff
@@ -15,19 +23,28 @@ const corsOptions = {
   credentials: true,
 };
 
+const SESSION_SECRET = process.env.SESSION_SECRET;
+
+if (!SESSION_SECRET) {
+  throw new Error('Missing SESSION_SECRET environment variable.');
+}
+
```

Comment:
Yeah okay. That seems like a much better implementation for this. I have reverted my changes, although it will reintroduce the Critical severity issue again, we can ignore it for now since it's getting fixed by the above PR.